parallel-install/pkg/deployment/deployment.go

```diff
@@ -150,6 +152,12 @@ InstallLoop:
 				i.processUpdateComponent(phase, cmp)
 				//Received a status update
 				if cmp.Status == components.StatusError {
+					// prerequisites fail fast
+					if phase == InstallPreRequisites {
+						err := errors.Wrapf(cmp.Error, "Error deploying prerequisite: %s", cmp.Name)
+						i.processUpdate(phase, ProcessExecutionFailure, err)
```

Comment:
maybe it would be also a good idea to log the error `i.cfg.Log.Error(err)`? since we are also doing this in the other cases

Reply:
The other cases are calling `i.logStatuses(statusMap)`, I will add that line.